### PR TITLE
webgpu: support step operator

### PR DESCRIPTION
**Pull request by xhcao**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7117)

### CODE REVIEW

File: tfjs-backend-webgpu/src/unary_op_webgpu.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class UnaryOpProgram implements WebGPUProgram {
   uniforms?: string;
   size = true;
 
-  constructor(outputShape: number[], op: UnaryOpType) {
+  constructor(outputShape: number[], op: UnaryOpType, extraConstParam = '') {
```
Comment (gyagp) [inline tfjs-backend-webgpu/src/unary_op_webgpu.ts:33]: As discussed, please pass in uniforms as a whole here

**Review by gyagp — COMMENTED** [review]
Please also correct the API definition using another PR.

**Review by gyagp — APPROVED** [review]
LGTM

**Comment by xhcao** [comment]
@qjia7 PTAL

File: tfjs-backend-webgpu/src/kernels/Step.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import {UnaryOpType} from '../unary_op_util';
+import {UnaryOpProgram} from '../unary_op_webgpu';
+
+export function step(
+    {inputs, attrs, backend}:
+        {inputs: UnaryInputs, attrs: StepAttrs, backend: WebGPUBackend}):
+    TensorInfo {
+  const {x} = inputs;
+  const program =
+      new UnaryOpProgram(x.shape, UnaryOpType.STEP, 'stepAlpha : f32,');
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/kernels/Step.ts:30]: Currently, for additional uniforms in unary ops, one example is [LeakyRelu](https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgpu/src/kernels/LeakyRelu.ts#L34). Should we keep them consistent in usage?

Comment (gyagp) [reply tfjs-backend-webgpu/src/kernels/Step.ts:30]: I think we can unify the LeakyRelu to this solution.

**Review by qjia7 — APPROVED** [review]

**Pull request merged**